mmengine/model/base_module.py

```diff
@@ -164,6 +164,8 @@ def _dump_init_info(self):
         # dump the information to the logger file if there is a `FileHandler`
         for handler in logger.handlers:
             if isinstance(handler, FileHandler):
+                if (handler.stream is None):
+                    continue
```

Comment:
Does it mean that if the `FIleHandler` has been closed, then `handler.stream` could be None and an error will be raised?

Reply:
Yes. NoneType occurred in my development environment.
I'll check the root cause of this problem again.